### DISCUSSION TITLE
Improve Developer Experience when working with multiple providers.

### DISCUSSION THREAD

**Discussion by fzn0x** [prisma/prisma#23843] [Q&A]
## Problem

Sometimes you don't know what database you currently on when you use SQLite provider with multiple clients.

```prisma
generator client {
  provider = "prisma-client-js"
  output   = "../../node_modules/@prisma-db/client"
}
```

For example, we change the output to `@prisma-db/client`, and after that we test it, and notice that why data is gone or lost.

Take a long time to realize that our client still using the old one.

Just because we forgot to change this:

```js
import { PrismaClient } from "@prisma/client";
```

to

```js
import { PrismaClient } from "@prisma-db/client";
```

We need to find a way to avoid this blunder. Because no errors or warnings from what we taste.

## Suggested solution

Generate the errors if developer modify the output of default schema and still use the client on code.

## Alternatives

Give warning that developer use old generated schema client?

**Comment by ludralph** [comment]
Hi @fzn0x 👋 

Thank you for raising this question.

Currently, Prisma Client does not provide a built-in warning or error system for this specific issue. However, you can open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your usecase so that our engineers can take a look.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
